Sources/ProcessEnv/Process+Parameters.swift

```diff
@@ -36,8 +36,13 @@ public extension Process {
         public func userShellInvocation() -> ExecutionParameters {
             let processInfo = ProcessInfo.processInfo
 
-            let shellPath = processInfo.shellExecutablePath
-            let args = ["-ilc", command]
+            #if !os(Linux)
+                let shellPath = processInfo.shellExecutablePath
+                let args = ["-ilc", command]
+            #else
+                let shellPath = "setsid"
+                let args = [processInfo.shellExecutablePath, "-ilc", command]
```

Comment:
What do you think about keeping shellPath the same for both platforms and just prepend that `setsid` into the args array?